keras_cv/models/object_detection/retinanet/retinanet_test.py

```diff
@@ -251,7 +251,10 @@ def test_call_with_custom_label_encoder(self):
         model(ops.ones(shape=(2, 224, 224, 3)))
 
     def test_tf_dataset_data_generator(self):
-        if backend.multi_backend() and keras.backend.backend() != "tensorflow":
+        if (
```

Comment:
You already handle this case in the `backend()` function you added to `config.py` above. Just import `config` and use `if config.backend() != "tensorflow"` here.

Example:
https://github.com/keras-team/keras-nlp/blob/e7bbf2b6c565f23a5be28557b60427d02223d091/keras_nlp/models/generative_task.py#L73